### PR TITLE
Sanitizes code reviews README and adds information about metrics

### PR DESCRIPTION
**Pull request by tompaana**
* Moves guidance solely focused on how to create PRs into its own file
* Sanitizes and restructures README.md
* Adds information on code reviews metrics
* Adds useful resources

Fixes issues #168 and #120.

### CODE REVIEW

**Comment by saroup** [comment]
I noticed that this PR was closed and reopened fresh without the context of old comments and conversations. I am wondering if this is best practice as you might be losing some context and having to switch back and forth to figure if something changed but it also keeps the review clean in terms of changes.
 Also, should we add something about it in the playbook? Like when to start fresh?

**Comment by tompaana** [comment]
> 
> 
> I noticed that this PR was closed and reopened fresh without the context of old comments and conversations. I am wondering if this is best practice as you might be losing some context and having to switch back and forth to figure if something changed but it also keeps the review clean in terms of changes.
> Also, should we add something about it in the playbook? Like when to start fresh?

There was a ghost filename conflict (due to case change in a filename) that would've been possibly to resolve, but I felt this was a cleaner solution. No history will be lost due to starting fresh.

File: code-reviews/README.md [hunk 1]
```diff
@@ -2,69 +2,54 @@
...
 
-#### Layers
+1. Add relevant reviewers:
+    * Add one or more reviewers (depending on your project's guidelines) to the PR. Ideally, you would add at least someone who has expertise and is familiar with the project or the language used
+    * Adding someone less familiar with the project or the language can aid in verifying the changes are understandable, easy to read, and increases the expertise within the team
+    * In CSE code-with projects with a customer team, it is important to include reviewers from both organizations for knowledge transfer
+      * In AzDO this can be enforced by setting groups as required reviewers, see [Automatically include code reviewers](https://docs.microsoft.com/en-us/azure/devops/repos/git/branch-policies?view=azure-devops#automatically-include-code-reviewers)
+1. Be open to receive feedback, discuss design/code logic and address all comments as follows:
+   * Resolve a comment, if the requested change has been made.
+   * Mark the comment as "won't fix", if you are not going to make the requested changes and provide a clear reasoning - "I'll do it later" is not an acceptible reason!
```
Comment (debugerr) [inline code-reviews/README.md]: If there is a comment out of scope of the current review / work item, we could add that adding a task for it and resolving as fix later would be an option.

Comment (tompaana) [reply code-reviews/README.md]: That is true, although it's a faux pas by reviewer. Will add.

File: code-reviews/README.md [hunk 2]
```diff
@@ -73,20 +58,20 @@ The goal of code reviews is to identify and remove defects before they can be in
...
 
-1. Understand the code you are reviewing  
-    * Read every line changed.  
-    * If we have a stakeholder review, it’s not necessary to run the PR unless it aids your understanding of the code.  
-    * AzDO orders the files for you, but you should read the code in some logical sequence to aid understanding.  
+1. Understand the code you are reviewing
+    * Read every line changed.
+    * If we have a stakeholder review, it’s not necessary to run the PR unless it aids your understanding of the code.
+    * AzDO orders the files for you, but you should read the code in some logical sequence to aid understanding.
     * If you don’t fully understand a change in a file because you don’t have context, click to view the whole file and read through surrounding code.
```
[the remark this answers is not in the document]
Comment (tompaana) [reply code-reviews/README.md]: Will clarify on line 65.

[the remark this answers is not in the document]
Comment (tompaana) [reply code-reviews/README.md]: Fixed

File: code-reviews/README.md (hunk 1 above)
Comment (tompaana) [reply code-reviews/README.md]: Fixed

**Pull request merged**